Documentation/DSCResources/VMHostSharedSwapSpace/VMHostSharedSwapSpace.md

```diff
@@ -0,0 +1,69 @@
+# VMHostSharedSwapSpace
+
+## Parameters
+
+| Parameter | Attribute | DataType | Description | Allowed Values |
+| --- | --- | --- | --- | --- |
+| **Server** | Key | string | The name of the Server we are trying to connect to. The Server can be a vCenter or ESXi. ||
+| **Credential** | Mandatory | PSCredential | The credentials needed for connection to the specified Server. ||
+| **Name** | Key | string | The name of the VMHost. ||
+| **DatastoreEnabled** | Optional | bool | Specifies if the Datastore option should be enabled or not. ||
+| **DatastoreName** | Optional | string | The name of the Datastore used by the Datastore option. ||
+| **DatastoreOrder** | Optional | long | The order of the Datastore option in the preference of the options. ||
+| **HostCacheEnabled** | Optional | bool | Specifies if the host cache option should be enabled or not. ||
+| **HostCacheOrder** | Optional | long | The order of the host cache option in the preference of the options. ||
```

Comment:
Same as above. Probably the meaning of order has to be described better for all properties

Reply:
Fixed.